circle.yml

```diff
@@ -1,3 +1,6 @@
+test:
+  override:
+    - npm test
```

Comment:
This was [already](https://circleci.com/gh/openfisca/openfisca-doc/73) inferred. Why add it explicitly?